test/vm-customize

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
+# This file is part of Cockpit.
+#
+# Copyright (C) 2013 Red Hat, Inc.
+#
+# Cockpit is free software; you can redistribute it and/or modify it
+# under the terms of the GNU Lesser General Public License as published by
+# the Free Software Foundation; either version 2.1 of the License, or
+# (at your option) any later version.
+#
+# Cockpit is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with Cockpit; If not, see <http://www.gnu.org/licenses/>.
+
+import argparse
+import os
+import sys
+import subprocess
+import re
+
+import testvm
+import testinfra
+
+parser = argparse.ArgumentParser(
+    description='Install Cockpit on a virtual machine')
+parser.add_argument('-v', '--verbose', action='store_true',
+                    help='Display verbose progress details')
+parser.add_argument('-I', '--install', action='append',
+                    dest="yumlist", default=[], help='Only upload and install')
```

Comment:
What about lowercase `i` to be consistent with the other parameters. Also, I would omit `Only`, since we could add both parameters.